src/components/main-form.js

```diff
@@ -50,6 +50,35 @@ option2.term = 30
 
 // CSS
 
+const Summary = styled(Row)`
+  b {
+    font-weight: 500;
```

Comment:
Do our `b`s need to look different in the app?

If this is the preferred weight, can we overwrite all the `b`s?

Reply:
I'm not sure if we should change all `b`s on the app. When simply using the `b` tag it looks too bold. I could take this out and make a styled component called `lb` or `LightBold` and use that instead if you prefer

Reply:
After some of my own testing, I do like `500` to be the global bold styling since it doesn't scream at the user 😄I'm not sure how to best do global styling though. Thoughts?

https://github.com/jgozal/moneycompass/pull/31/files#diff-13bae51262f519caebe5af0bd2fcb33cR56